deploy/client-certificate-ds.yaml

```diff
@@ -130,6 +133,9 @@ spec:
       - name: etcfs
         hostPath:
           path: /etc
+      - name: minidumps-dir
+        hostPath:
+          path: /var/lib/quobyte/.minidumps
```

Comment:
Who creates this path?

Reply:
Docker creates the source of a volume mapping if it does not exist.